perl6intro.adoc

```diff
@@ -3224,3 +3224,5 @@ NOTE: For more info on the Native Calling Interface, see https://docs.perl6.org/
 * link:http://pl6anet.org[pl6anet] blog aggregator. Stay tuned by reading blog posts that focus on Perl 6.
 
 * link:https://www.reddit.com/r/perl6/[/r/perl6] Subscribe to the Perl 6 subreddit.
+
+* link:https://alerts.perl6.org[P6lert] alerts from Perl 6 Core Developers. Keeping up to date with important changes.
```

Comment:
This is possibly a typo; "Keep" is better.

Reply:
I've fixed it, I was using the description from the Perl 6 Alerts.